backend/Tasque.BLL/Services/TaskService.cs

```diff
@@ -39,7 +40,7 @@ public async Task<TaskDto> CreateTask(TaskDto model)
             var entity = _mapper.Map<Common.Entities.Task>(model);
             var key = _dbContext.Projects.FirstOrDefault(p => p.Id == model.ProjectId)?.Key;
             var count = _dbContext.Tasks.Where(t => t.ProjectId == model.ProjectId).Count();
-            entity.Key = key + '-' + count;
+            entity.Key = key + '-' + count++;
```

Comment:
Please, don't use count... If user moves one task we can have two tasks with same key (And better use numeric column for task number

Reply:
Done